descope/management/tenant.py

```diff
@@ -11,6 +11,8 @@ def create(
         id: Optional[str] = None,
         self_provisioning_domains: Optional[List[str]] = None,
         custom_attributes: Optional[dict] = None,
+        enforce_sso: Optional[bool] = None,
```

Comment:
you can set it as False/True by default (instead of None) so it will be clear what the default is..